lib/pages/chat/chat.dart

```diff
@@ -1478,6 +1431,19 @@ class ChatController extends State<Chat>
     }
   }
 
+  void handlePopBackFromPinnedScreen(Object? popResult) async {
+    Logs().d(
+      "PinnedEventsController()::handlePopBack(): popResult: $popResult",
+    );
+    if (popResult is Event) {
+      await TwakeDialog.showFutureLoadingDialogFullScreen(
+        future: () => scrollToEventIdAndHighlight(popResult.eventId),
+      );
```

Comment:
I don't know why we need to await and loading. Apply the same new logic to jump message?

Reply:
scrollToEventId is async -> scrollToEventIdAndHighlight is async and because we jump not very fast, i think put it in loading for prevent user click other things

Reply:
But this will wait for the highlight end, then the dialog will disappear. Remove it makes better UX